servers/graphql-kotlin-server/src/main/kotlin/com/expediagroup/graphql/server/execution/GraphQLContextFactory.kt

```diff
@@ -30,10 +29,3 @@ interface GraphQLContextFactory<out Context : GraphQLContext, Request> {
      */
     fun generateContext(request: Request): Context?
 }
-
-/**
- * Default context factory that generates GraphQL context with empty concurrent map that can store any elements.
- */
-class DefaultGraphQLContextFactory<T> : GraphQLContextFactory<DefaultGraphQLContext, T> {
```

Comment:
This class was not used anywhere. Each server can provide their own default implmentation